Maestro/subscriptions.json

```diff
@@ -1254,6 +1254,11 @@
         "https://github.com/aspnet/EntityFramework6/blob/release/3.0-preview3/**/*",
         "https://github.com/aspnet/EntityFrameworkCore/blob/release/3.0-preview3/**/*",   
         "https://github.com/aspnet/Extensions/blob/release/3.0-preview3/**/*",
+        "https://github.com/aspnet/AspNetCore/blob/release/3.0-preview4/**/*",
+        "https://github.com/aspnet/AspNetCore-Tooling/blob/release/3.0-preview4/**/*",
+        "https://github.com/aspnet/EntityFramework6/blob/release/3.0-preview4/**/*",
+        "https://github.com/aspnet/EntityFrameworkCore/blob/release/3.0-preview4/**/*",   
+        "https://github.com/aspnet/Extensions/blob/release/3.0-preview4/**/*",
```

Comment:
Why leave the old ones (just above) around?

Also, why haven't we seen the expected aspnet/AspNetCore `release/3.0-preview4` -> `master` PRs? I had to do a manual merge (/aspnet/AspNetCore-Tooling#401) earlier today.

Reply:
Is the repo's [Maestro web hook](https://github.com/dotnet/core-eng/blob/master/Documentation/Maestro/web-hooks.md) set up? dotnet-maestro-bot [has never submitted anything to that repo](https://github.com/aspnet/AspNetCore-Tooling/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3Adotnet-maestro-bot).

Reply:
Good question. I'll look later&hellip;

Reply:
> Why leave the old ones (just above) around?

The branches are still present in the repos. Once we delete those branches (which I think we should do now), i'll clean out the old subscriptions.

>  why haven't we seen the expected aspnet/AspNetCore release/3.0-preview4 -> master PRs?

Because I haven't merged this PR... yet which tells the bot to do this :)